offlinevalidator/offlinevalidator.go

```diff
@@ -448,6 +412,84 @@ func sliceContains(slice []string, match string) bool {
 	return false
 }
 
+func mapContainsNil(mp map[string]interface{}) bool {
+	for _, m := range mp {
+		if m == nil {
+			return true
+		}
+	}
+	return false
+}
+
+func sliceContainsNil(slice []interface{}) bool {
+	for _, s := range slice {
+		if s == nil {
+			return true
+		}
+	}
+	return false
+}
+
+// We check if the element is non-string, non-float64, non-boolean. Then it is another node or <nil>. There is no other option.
+func isNonSFB(v interface{}) bool {
```

Comment:
I'm strongly against such acronyms, let's unwind it.

Reply:
Changed to `isNonStringFloatBool`.